### PR TITLE
Sgreenberg fix national districts

### PR DESCRIPTION
**Pull request by sgreenberg**

### CODE REVIEW

**Comment by sgreenberg** [comment]
Not ready to review yet. Let's get @bobg's pr (https://github.com/SisterDistrictTech/db-scripts/pull/14) merged first. With that merged, the diff against master for this PR will just be one commit.

File: SD_DB_Setup.sql
Comment (nsudarsanam) [inline SD_DB_Setup.sql:5]: @LinneaHarts fyi

File: sdlib/national_districts.py
Comment (nsudarsanam) [inline sdlib/national_districts.py:455]: this is the only really big change from bob's PR right? the merge is going to get interesting :)

**Comment by sgreenberg** [comment]
@bobg, @nsudarsanam - I rebased on top of master. Take a look at the PR. Once you both approve, I'll merge. The changes to national_districts.py were made following the pattern that Bob established for pres_races.

File: sdlib/national_districts.py
Comment (nsudarsanam) [inline sdlib/national_districts.py:16]: we probably want a comment somewhere of how we are treating "at-large" districts?

**Review by nsudarsanam — APPROVED** [review]

File: sdlib/national_districts.py
Comment (sgreenberg) [reply sdlib/national_districts.py:16]: Sure thing. I added it here - https://docs.google.com/document/d/1KyW6t-lnKBAEi_fQykHuJ2ZYfaFj_8gyNTrZmMnKFcQ/edit#heading=h.653ok8drsfcf. 

As long as we have a database document, that seems like the place for such explanation.

**Review by bobg — APPROVED** [review]

**Comment by bobg** [comment]
LGTM

**Comment by LinneaHarts** [comment]
Looks good to me

**Comment by sgreenberg** [comment]
got the approval from Linnea in slack.

**Pull request merged**
